pulumi/bin/copy_artifacts_from_rc.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 # This will grab a subset of keys from `origin/rc`'s Pulumi.testing.yaml.
 set -euo pipefail
+set -o xtrace
```

Comment:
This script has proven to be somewhat brittle, so I'd actually like to leave this option on for future users, if that's alright!